### PR TITLE
fix get queries field type

### PR DESCRIPTION
**Pull request by tarunvelli**

### CODE REVIEW

File: protoc-gen-graphql/template.go [hunk 1]
```diff
@@ -253,7 +253,7 @@ func (x *graphql__resolver_{{ $service.Name }}) GetQueries(conn *grpc.ClientConn
 			Args: graphql.FieldConfigArgument{
 			{{- range .Args }}
 				"{{ .FieldName }}": &graphql.ArgumentConfig{
-					Type: {{ .FieldType $.RootPackage.Path }},
+					Type: {{ .FieldTypeInput $.RootPackage.Path }},
```
Comment (ysugimoto) [inline protoc-gen-graphql/template.go:256]: This section will be used for Query parameter generation so I think it should keep using `.FieldType`.

**Review by ysugimoto — COMMENTED** [review]
@tarunvelli Could you show me before/after execution code with simple proto file?
This section is used for `Query` argument, not an input I think.
Currently, I don't see any problem with this section but probably I made a bit bug on #29

**Comment by tarunvelli** [comment]
@ysugimoto I have added my observations here https://gist.github.com/tarunvelli/5cb0e12eeb12df29fa0bcb20aab54f4e

In short, the graphql gateway isn't able to read the input value when using `FieldType` in query argument generation

**Comment by tarunvelli** [comment]
@ysugimoto any update on this?

**Comment by ysugimoto** [comment]
@tarunvelli Sorry for being late. I looked at your Gist, thanks.

I understand your point about this PR. I guess it caused by `ptypes` implementation on [here](https://github.com/ysugimoto/grpc-graphql-gateway/tree/master/ptypes/wrappers).

I intend to use `FieldType` and `FieldTypeInput` for definitely distinguish between `type` and `input` definition in GraphQL.

Your change seems to look fine, but I'm still wondering why the query is successful when using `FieldTypeInput`,  and failure on `FieldType`, so I'll continue to investigate `graphql-go`'s implementation and `ptypes` implementation.

If you have any information about it. let me know, thanks!

**Comment by ysugimoto** [comment]
@tarunvelli I've looked into it deeply and I realized your changes are a good way to fix the problem!

The primal problem is that is not able to pass an object to the Query argument. The reason why is I've planned to extract protobuf's message and list fields of them, and I didn't consider of `message` type argument.

Hence, it causes not only types like `google.protobuf.Int32Value` but also any nested message. I reproduced as following:

```protobuf
syntax = "proto3";

import "graphql.proto";
import "google/protobuf/wrappers.proto";

option go_package = ".;echo";

service Echo {
  // gRPC service information
  option (graphql.service) = {
    host: "localhost:50051"
    insecure: true
  };

  rpc Sample(SampleMessage) returns (SampleMessage) {
    option (graphql.schema) = {
      type: QUERY
      name: "sample"
    };
  }
}
message Member {
  int64 id = 2;
}

message SampleMessage {
  Member member = 1;
}
```

This example is similar to your PR, the difference is that using the user-defined `message` type as query argument field and this case also responds `null` for `member` field.

To solve this problem, we need to modify a bit in `protoc-gen-graphql` process but I have a plan to do it. 

Totally you're right. Nice catch and thank you for making PR and giving me an example!

**Review by ysugimoto — APPROVED** [review]
LGTM 👍

File: protoc-gen-graphql/generator/generator.go [hunk 2]
```diff
@@ -344,7 +344,7 @@ func (g *Generator) analyzeService(f *spec.File, s *spec.Service) error {
 func (g *Generator) analyzeQuery(f *spec.File, q *spec.Query) error {
 	g.logger.Write("package %s depends on query request %s", f.Package(), q.Input.FullPath())
 	q.Input.Depend(spec.DependTypeMessage, f.Package())
-	if err := g.analyzeFields(f.Package(), q.Input, q.PluckRequest(), false, false); err != nil {
```
Comment (ysugimoto) [inline protoc-gen-graphql/generator/generator.go:347]: Ah, sorry this change does not need. I have a plan to fix more problems globally. I appreciate it you revert this line.

Comment (tarunvelli) [reply protoc-gen-graphql/generator/generator.go]: removed it from the current pr 👍

**Comment by ysugimoto** [comment]
@tarunvelli I've just published a fixed version as `v0.19.0`. I'm happy if you could try this version 😄

**Pull request merged**